### PR TITLE
Update EmailAttestationRequired

### PR DESCRIPTION
**Pull request by salarson**

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -1726,6 +1726,8 @@ Accept wildcard characters: False
 
 Sets email attestation to required.
 
+If people who use a verification code have selected to "stay signed in" in the browser, they must prove they can still access the account they used to redeem the sharing invitation. You can set the number of days for email attestation with -EmailAttestationReAuthDays. This setting affects only ad-hoc external recipients
```
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:1729]:
```suggestion
If people who use a verification code have selected to "stay signed in" in the browser, they must prove they can still access the account they used to redeem the sharing invitation. You can set the number of days for email attestation with -EmailAttestationReAuthDays. This setting affects only ad-hoc external recipients.
```

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]
I am ok once @dariomws suggestion is applied.

**Review by tseward — APPROVED** [review]
Approved with dariomws' changes.

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md (hunk 1 above)
Comment (tiburd) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:1729]:
```suggestion
If people who use a verification code have selected to "stay signed in" in the browser, they must prove that they can still access the account that they previously used to redeem the sharing invitation. You can set the number of days for email attestation with **-EmailAttestationReAuthDays**. This setting affects only ad-hoc external recipients
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 2]
```diff
@@ -1726,6 +1726,8 @@ Accept wildcard characters: False
 
 Sets email attestation to required.
 
+If people who use a verification code have selected to "stay signed in" in the browser, they must prove they can still access the account they used to redeem the sharing invitation. You can set the number of days for email attestation with -EmailAttestationReAuthDays. This setting affects only ad-hoc external recipients.
```
Comment (tiburd) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:1729]:
```suggestion
If people who use a verification code select to "stay signed in" in the browser, they must prove that they can still access the same account that they used to redeem the sharing invitation. You can set the number of days for email attestation with **-EmailAttestationReAuthDays**. This setting affects only ad-hoc external recipients.
```

**Pull request merged**